Makefile

```diff
@@ -29,8 +29,18 @@ IMAGE_ORG ?= $(USER)
 # IMAGE_TAG_BASE defines the namespace and part of the image name for remote images.
 IMAGE_TAG_BASE ?= quay.io/${IMAGE_ORG}/network-observability-console-plugin
 
+# Standalone `true` is used to build frontend outside of OCP Console environment
+STANDALONE ?= false
+BUILDSCRIPT = 
+
+ifeq (${STANDALONE}, true)
+	BUILDSCRIPT := :standalone
+	IMAGE_TAG_BASE := quay.io/${IMAGE_ORG}/network-observability-standalone-frontend
+endif
```

Comment:
@jotak @OlivierCazade are you fine with that approach ?

I don't want to maintain 2 separate docker files so I feel it would be better.

Reply:
here we go https://github.com/netobserv/network-observability-console-plugin/pull/325/commits/7518bfc6806a41cf0bf3cf074dbecf3bcb6a3b7b

Reply:
thanks!